app/src/components/ProgressBar.tsx

```diff
@@ -0,0 +1,58 @@
+import { useTheme } from 'aries-bifold'
+import React, { useEffect, useState } from 'react'
+import { StyleSheet, View, LayoutChangeEvent, Animated } from 'react-native'
+
+export interface ProgressBarProps {
+  progressPercent: number
+}
+
+const ProgressBar: React.FC<ProgressBarProps> = ({ progressPercent }) => {
+  const { ColorPallet } = useTheme()
+  const [progressBarWidth, setProgressBarWidth] = useState(700)
```

Comment:
From my understanding this is used to position the progress bar animations right? We might want to use % values instead of pixels, not sure how it will scale to bigger screens like tablets.

Reply:
Concur. Can we use the `width` value from RN dimensions?

Reply:
I had the width being set from the outer containers `onLayout` in case we wanted to use this component somewhere where the width wouldn't be the total screen width, but we probably won't, so I've simplified it to just use the window `width`